### DISCUSSION TITLE
Please fix bug game Story Of the season and please support this emulator with keyboard

### DISCUSSION THREAD

**Discussion by Nerolya** [skyline-emu/skyline#1536] [Q&A] [answered]
Please Support This Emulator with keyboard🙏
And please fix the blur graphics display bug in the game Story Of The Season: Friends Of Mineral Town
I really like this emulator I always support this project👍🔥🔥🔥❤️❤️

**Comment by dima-xd** [comment, marked as answer]
All of the above will be fixed in a future
